numpy/ctypeslib.py

```diff
@@ -119,6 +119,10 @@ def load_library(libname, loader_path):
         OSError
             If there is no library with the expected extension, or the
             library is defective and cannot be loaded.
+        Examples
+        ------
+        >>> np.ctypeslib.load_library('_rational_tests', np.core.__file__)
+        <CDLL '/home/ankur/.local/lib/python3.10/site-packages/numpy/core/_rational_tests.cpython-310-x86_64-linux-gnu.so', handle 55675a3a97f0 at 0x7f75d5ccf700>
```

Comment:
Hi @ankur0904 - I think if we want to add examples here, I'd go for some explanation of what the input arguments are. As it is, the example doesn't add a lot to the function definition. But I also can't think of a better example... We should also make sure the output is reasonable for our doctests.

Reply:
IMO adding examples doesn't improve the docstring. I'd vote to close this and close #22268 as resolved.

Reply:
@melissawm @rossbar Thanks for the feedback.

Reply:
Should we need to close the PR?

Reply:
@ankur0904 Please do. Feel free to pick up another issue!

Reply:
Okay